### PR TITLE
Adds contribution guidelines to VueUI

### PR DESCRIPTION
**Pull request by Karolis2011**
This needs to be added, because ***documentation***.

### CODE REVIEW

**Comment by Karolis2011** [comment]
!review

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -219,3 +219,24 @@ for (var/client/C in world)
 All tables for the database should be prefixed according to the following list:
 * `ss13_` for tables in which ingame data is held.
 * `discord_` for tables in which BOREALIS data is held.
+
+# HTML UI Standards
+
+### UI conversion policy
+Due to our current situation with 5 different HTML UI systems we are now enforcing new policy that all new uis without exception should be made in VueUi ui system. This policy also applies to editing existing uis, but there is few exceptions:
```
Comment (skull132) [inline .github/CONTRIBUTING.md]:
```suggestion
Due to our current situation with 5 different HTML UI systems we are now enforcing a policy that all new UIs should be made using the VueUi UI system. This policy also applies to editing existing UIs, with the following exceptions:
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -219,3 +219,24 @@ for (var/client/C in world)
...
+
+ 1. Modification is security / severe bug fix.
+ 0. It is typo fix.
+ 0. Touched UI file is too large.
+ 0. VueUi can't accommodate that type of UI.
+
+### Responsiveness
```
Comment (skull132) [inline .github/CONTRIBUTING.md]: whaddoes this mean?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -219,3 +219,24 @@ for (var/client/C in world)
...
+All new UIs must be responsive, that means that when parameters change in game world, UI data must update as quickly as possible to reflect that change. If change is time dependant, then client side time approximation should be used.
+
+### Conditional usage policy
+If you need to use conditional rendering inside UI, then try to put conditional statements on elements you want to hide, then try using `<template>` to apply condition to multiple components.
+
+For conditional rendering try to prefer to use `v-show` attribute when change is expected to be often occurring. Use `v-if` when you need `v-else` and switch is expected expected not often.
+
+### Reusability
+If there is segment of ui that is used multiple times with different content. then we strongly encourage making of new component. If that component is general or may be reused globally, then it should be made in to global component (placed in `vui` folder), else it should be made in to ui specific component that must be placed in folder inside `view` folder.
```
Comment (skull132) [inline .github/CONTRIBUTING.md]:
```suggestion
If there is segment of UI that is used multiple times with different content, then we strongly encourage making of new component. If that component is general or may be reused globally, then it should be made in to global component (placed in `vui` folder), else it should be made in to UI specific component that must be placed in folder inside `view` folder.
```

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (Karolis2011) [reply .github/CONTRIBUTING.md]: It means that uis should not require you to click refresh.

**Pull request merged**